nilearn/datasets/atlas.py

```diff
@@ -901,12 +1003,22 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, keys are:
 
-        - "maps": str. path to nifti file containing regions.
-
-        - "labels": list of the names of the regions
+            - 'maps': :obj:`str`, path to nifti file containing the
+              regions. The image has shape ``(91, 109, 91)`` and contains
+              117 unique integer values defining the parcellation.
+            - 'labels': :obj:`list` of :obj:`str`, list of the names of the
+              regions. This list has 116 names as 'Background' (label 0) is
+              not included in this list.
+            - 'indices': :obj:`list` of :obj:`str`, indices mapping 'labels'
+              to values in the 'maps' image.
+              Since the values in the 'maps' image are not consecutive, the
```

Comment:
here also rather than are not consecutive, "do _not_ correspond to indices in `labels`, but rather to values in `indices`"